src/create-jobs.js

```diff
@@ -8,19 +8,31 @@ import create_additional_jobs from './additional-checks.js';
 import validateAndNormalizeChecks from './validate-and-normalize-checks-from-config.js';
 import parseJsonFile from "./json.js";
 
+const xmlSchema = {
+    phpunit: 'vendor/phpunit/phpunit/phpunit.xsd',
+    phpcs: 'vendor/squizlabs/php_codesniffer/phpcs.xsd',
+    psalm: 'vendor/vimeo/psalm/config.xsd',
+};
+
 /**
  * @param {String} filename
- * @return {Boolean}
+ * @return {function(): Boolean}
```

Comment:
Is this correct? fs.existsSync is synchronous, so we'd just get back the return value, wouldn't we?

Reply:
Never mind... just saw we've ALWAYS returned a function that returns boolean. :blush:

Reply:
Yeah, there are a couple of fixes I made to the doc types in this manner just because I noticed them.